### ISSUE TITLE
Synthetic NEWS vignette

### ISSUE THREAD

**Issue by tomjemmett** [nhs-r-community/NHSRdatasets#55]
Currently, in the synthetic news data vignette, the data is loaded [directly from a url](https://github.com/nhs-r-community/NHSRdatasets/blob/82304f9bc6750fe5d5d8347db687232cfbd92d68/vignettes/synthetic_news_data.Rmd#L52).

This is a potential issue for rendering with CRAN: occasionally file downloads can fail, which will cause the vignette to error. If this happens, CRAN can archive the package.

If the vignette is to remain as it is, this file should be directly included within the package (in the inst/ directory).

However, I think this vignette is a little out of place for this package. The other vignettes detail the source of the data, and then simple examples of how to use the data. This vignette instead explains how to generate the data, which feels out of scope for this package (there is already a [blog post](https://nhsrcommunity.com/blog/can-we-rely-on-synthetic-data-to-overcome-data-governance-issue-in-healthcare/) on the subject), and nothing about useful ways of using this data for learning purposes.

**Comment by Lextuga007** [comment]
Agreed, there are also a lot of package dependencies because of the vignettes. We need simpler vignettes and these can also be linked to the NHSR courses. 

The vignettes currently could be moved to the repo [Demos and How to](https://github.com/nhs-r-community/demos-and-how-tos) so they don't get lost. Then perhaps link to them from here or is that a similar issue with CRAN? Would it be on the README?

**Comment by Lextuga007** [comment]
Looking at the vignette it is synthesising already synthesised data so it works really well as a blog. The question though is how useful is the data itself in the package? I'm not able to create a vignette using it as it's not data I'm familiar with and it's likely it has been cleaned for statistical use as the column `male` is 1 or 0. I think the package is better suited to data that is closer to the original, where possible, to help with familiarity.

Question: should the data be removed?
